modules/default-secrets.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="default-secrets_{context}"]
+= Default secrets
+
+By default, {product-title} creates the following secrets:
+
+* A `builder` dockercfg secret
+* A `builder` service account token secret
+* A `default` dockercfg secret
+* A `default` service account token secret
+* A `deployer` dockercfg secret
+* A `deployer` service account token secret
+
+// TODO: Do we need to describe what each of these are used for? Are the ALL used for accessing the internal registry? Or something else?
```

Comment:
@sanchezl Is the general statement below about these secrets being used to access the internal registry good, or is there some more info we could expand that with? Or give more detail for each of these 6 specifically?

Reply:
This needs to be a general statement, I provided an alternate text suggestion in another comment.